src/js/components/ServiceDetail.js

```diff
@@ -30,7 +37,10 @@ class ServiceDetail extends mixin(InternalStorageMixin, TabsMixin) {
 
     this.state = {
       currentTab: Object.keys(this.tabs_tabs).shift(),
-      isServiceFormModalShown: false
+      isServiceFormModalShown: false,
+      isServiceDestroyConfirmShown: false,
+      isServiceScaleFormModalShown: false,
+      isServiceSuspendConfirmShown: false
```

Comment:
There is always only one modal open, right? How about one variable holding a modal id or nothing to toggle all these modal states?